src/overlays/actors/ovl_Door_Spiral/z_door_spiral.c

```diff
@@ -31,30 +40,223 @@ const ActorInit Door_Spiral_InitVars = {
     (ActorFunc)DoorSpiral_Init,
     (ActorFunc)DoorSpiral_Destroy,
     (ActorFunc)DoorSpiral_Update,
-    (ActorFunc)DoorSpiral_Draw
+    (ActorFunc)DoorSpiral_Draw,
+};
+
+typedef struct {
+    /* 0x0 */ Gfx* unk_00[2];
+    /* 0x8 */ u32 unk_08;
+    /* 0x9 */ u8 unk_09;
+    /* 0xA */ u8 translateZ;
+    /* 0xB */ u8 unk_0B;
+    /* 0xC */ u8 unk_0C;
+} SpiralInfo; // size = 0x10
+
+SpiralInfo sSpiralInfoTable[] = {
+    { NULL, NULL, 0, 130, 12, 50, 15 },
+    { gameplay_dangeon_keep_DL_0219E0, gameplay_dangeon_keep_DL_01D980, 0, 130, 12, 50, 15 },
```

Comment:
```suggestion
    { { NULL, NULL }, 0, 130, 12, 50, 15 },
    { { gameplay_dangeon_keep_DL_0219E0, gameplay_dangeon_keep_DL_01D980 }, 0, 130, 12, 50, 15 },
```
Since the Display lists are stored in an array, could you update the struct?